src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -96,6 +94,38 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return filter.positiveAttributeFilter ? false : true;
+    }
+
+    private getIdentifierAndUri() {
```

Comment:
It can never be both identifier and uri, why not get the whole qualifier?
```
return identifier ? { identifier } : { uri };
```

Reply:
Done.